static/app/views/settings/organizationMembers/inviteBanner.spec.tsx

```diff
@@ -239,23 +215,17 @@ describe('inviteBanner', function () {
       body: {data: promptResponse},
     });
 
-    await act(async () => {
-      await render(
-        <InviteBanner
-          onSendInvite={() => {}}
-          organization={org}
-          allowedRoles={[]}
-          onModalClose={() => {}}
-        />
-      );
-    });
+    const {container} = render(
+      <InviteBanner
+        onSendInvite={() => {}}
+        organization={org}
+        allowedRoles={[]}
+        onModalClose={() => {}}
+      />
+    );
 
-    expect(mockPrompt).toHaveBeenCalled();
-    expect(
-      screen.queryByRole('heading', {
-        name: 'Bring your full GitHub team on board in Sentry',
-      })
-    ).not.toBeInTheDocument();
```

Comment:
checking for exact text to not be in the document is not as good as checking that the component rendered nothing since the text might change and the test will pass.